src/main/webapp/js/jquery.fancybox-1.3.4.js

```diff
@@ -577,8 +577,8 @@
 
 		_finish = function () {
 			if (!$.support.opacity) {
-				content.get(0).style.removeAttribute('filter');
-				wrap.get(0).style.removeAttribute('filter');
+				content.css('filter', '');
+				wrap.css('filter', '');

```

Comment:
Can you test this to see if it works? If so I think it would be less risky by keeping the form of the original code.

```suggestion
				content.get(0).style.removeProperty('filter');
				wrap.get(0).style.removeProperty('filter');
```

Reply:
@basil 

Confirming the above works fine as well